pkgs/applications/audio/drumgizmo/default.nix

```diff
@@ -7,7 +7,7 @@ stdenv.mkDerivation rec {
   name = "drumgizmo-${version}";
 
   src = fetchurl {
-    url = "http://www.drumgizmo.org/releases/${name}/${name}.tar.gz";
+    url = "https://www.drumgizmo.org/releases/${name}/${name}.tar.gz";
```

Comment:
But there is still the issue of generating the hash the first time (e.g., on version bumps), where https is potentially better (assuming not all maintainers verify the source by using a separate cryptographic signature or some other means).